src/target/riscv/riscv.c

```diff
@@ -3568,6 +3686,13 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.help = "Control dcsr.ebreaku. When off, U-mode ebreak instructions "
 			"don't trap to OpenOCD. Defaults to on."
 	},
+	{
+		.name = "itrigger",
+		.handler = riscv_itrigger,
+		.mode = COMMAND_ANY,
+		.usage = "set [vs] [vu] [nmi] [m] [s] [u] <mie bits>|clear",
+		.help = "Set or clear a single itrigger."
```

Comment:
Little favor to those who may not be familiar with the term `itrigger`:

```suggestion
		.help = "Set or clear a single itrigger (RISC-V Interrupt Trigger)."
```

Reply:
How about "Set or clear a single interrupt trigger."